cmd/argocd/commands/login.go

```diff
@@ -70,22 +70,25 @@ argocd login cd.argoproj.io --core`,
 				server = "kubernetes"
 			} else {
 				server = args[0]
-				dialTime := 30 * time.Second
-				tlsTestResult, err := grpc_util.TestTLS(server, dialTime)
-				errors.CheckError(err)
-				if !tlsTestResult.TLS {
-					if !globalClientOpts.PlainText {
-						if !cli.AskToProceed("WARNING: server is not configured with TLS. Proceed (y/n)? ") {
-							os.Exit(1)
+
+				if !globalClientOpts.SkipTestTLS {
```

Comment:
This is awesome! Two suggestions:

1. Does this need to be a global config option, or can we _just_ add it to `login`?
2. How would you feel about adding `(this can help when the command hangs for no apparent reason)` or something like that so people can discover the option without finding the related GitHub issue?

Reply:
Hi, you are completely right:

1. moved the new flag to the login sub-command
2. added the hint about when this flag might help